schema.json

```
@@ -63,87 +69,362 @@
+      "type": "date",
+      "example": "2017-06-27",
+      "constraints": {
+        "required": false
+      }
+    },
+   
+    {
+      "name":"anneDepotDemande",
+      "title": "Année de dépôt dossier",
+      "description": "Année de dépôt de la demande de subvention.",
+      "type": "year",
+      "example": "2017",
+      "constraints": {
+        "required": false
+      }
+    },
+
+    {
+      "name": "dispositifAide",
+      "title": "Dispositif d'aide",
+      "description": "Nom du dispositif de financement ou du programme de subvention.",
+      "type": "string",
+      "examples": "FIPDR, MILDECA, Fond sociale européen...",
+      "constraints": {
+        "required": false
+      }
+    },
+
+    {
+      "name": "sousDispositif",
+      "title": "Sous-dispositif de subvention",
+      "description": "s'il y en a, indiquer le nom du sous-dispositif ou du sous-programme.",
+      "type": "string",
+      "constraints": {
+        "required": false
       }
     },
+
+    {
+      "name": "statutLabel",
+      "title": "Statut de la demande de subvention",
+      "description": "Statut du traitement de la demande. Valeurs possibles : en cours d'instruction, attribuée, inéligible (la subvention ne respecte pas les critères d'éligibilité. Exemple : dossier incomplet), refusée (la demande a été instruite mais la subvention n'a pas été accordée).",
+      "type": "string",
+      "constraints": {
+        "required": false,
+        "enum": [
+          "en cours d'instruction",
+          "attribuée",
+          "inéligible",
+          "refusée"
+        ]
+      }
+    },
+
     {
       "name": "objet",
       "title": "Objet de la subvention",
-      "description": "Description de l'objet de la subvention attribuée limitée à 256 caractères maximum.",
+      "description": "Renseigner 'fonctionnement global' si la subvention est destinée au fonctionnement globale ou la liste des projet(s)/action(s) si la subvention est destinée à un ou plusieurs projets/actions",
       "type": "string",
       "example": "Animations climat-énergie dans les lycées de la région",
       "constraints": {
         "required": true,
         "maxLength": 256
       }
     },
+    {
+      "name": "nature",
+      "title": "Nature de la subvention",
+      "description": "Plusieurs choix possibles en combinant les valeurs 'aide en numéraire' et/ou 'aide en nature'. Les valeurs autorisées sont 'aide en numéraire', 'aide en nature', 'aide en numéraire;aide en nature', 'aide en nature;aide en numéraire'. Quand la nature de la subvention est à la fois en numéraire et en nature, le signe de séparation des valeurs est le point-virgule.",
+      "type": "string",
+      "example": "aide en numéraire;aide en nature",
+      "constraints": {
+        "required": true,
+        "enum": [
+          "aide en numéraire",
+          "aide en nature",
+          "aide en numéraire;aide en nature"
```

Le ";" est parfois utilisé comme séparateur des colonnes, mieux vaut ne pas l'utiliser ici : 
```suggestion
          "aide en numéraire|aide en nature"
```
ou (je préfère)
```suggestion
          "aide en numéraire et en nature"
```